BusinessModels/GeneSIS.md

```diff
@@ -34,7 +34,7 @@ This page contains the business model for the GeneSIS (Orchestration and deploym
     <td rowspan="1">
       <p>
         
-Current solutions: (i) No deployment solution tailored for IoT, Edge, and Cloud infrastructure, (ii) No monitoring of the execution flow of the application. This is particularly relevant for debugging on IoT devices (as this is typically a complicated task), (iii) deployment tool do not integrate with security and privacy tools.
+Current solutions: (i) No deployment solution tailored for IoT, Edge, and Cloud infrastructure (in particular no support for deployment on devices with no direct access to Internet), (ii) No monitoring of the execution flow of the application. This is particularly relevant for debugging on IoT devices (as this is typically a complicated task), (iii) deployment tool do not integrate with security and privacy tools, and (iv) no customizable solution for rolling and continuous deployment.
```

Comment:
Current solutions: (i) No deployment solution tailored for IoT, Edge, and Cloud infrastructures (in particular no support for deployment on devices with no direct access to Internet), (ii) No monitoring of the execution flow of the application. Monitoring is particularly relevant for debugging on IoT devices (as this is typically a complicated task), (iii) deployment tools do not integrate with security and privacy tools, and (iv) no customizable solution for rolling and continuous deployment.